hypervisor/arch/x86/cpu_state_tbl.c

```diff
@@ -76,16 +76,16 @@ static const struct cpu_state_table {
 	struct cpu_state_info	state_info;
 } cpu_state_tbl[] = {
 	{"Intel(R) Atom(TM) Processor A3960 @ 1.90GHz",
-		{ARRAY_SIZE(px_a3960), px_a3960,
-		 ARRAY_SIZE(cx_a3960), cx_a3960}
+		{(uint8_t)ARRAY_SIZE(px_a3960), px_a3960,
```

Comment:
Is ARRAY_SIZE(x) function like API? If yes, let us fix it before this change.

Reply:
#define ARRAY_SIZE(arr) (sizeof(arr) / sizeof((arr)[0]))
arr's type can be any type,use function to replace can't suitable for this situation.